xbmc/cores/VideoRenderers/OverlayRendererGUI.cpp

```diff
@@ -125,7 +125,12 @@ COverlayText::COverlayText(CDVDOverlayText * src)
   }
   else
   {
-    m_align  = ALIGN_VIDEO;
+    if(m_subalign == SUBTITLE_ALIGN_TOP_INSIDE
+    || m_subalign == SUBTITLE_ALIGN_TOP_OUTSIDE)
```

Comment:
you would save me a merge conflict if you copied the code from my branch. every time I hit this stupid formatting style I change it.